Document/0x05b-Basic-Security_Testing.md

```diff
@@ -279,102 +279,19 @@ Frida also provides bindings for various languages, including Python, C, NodeJS,
 
 ##### Drozer
 
--- ToDo: <https://github.com/OWASP/owasp-mstg/issues/1233>
+[Drozer](https://github.com/mwrlabs/drozer "Drozer on GitHub") is an Android security assessment framework that allows you to search for security vulnerabilities in apps and devices by assuming the role of a third-party app interacting with the other application's IPC endpoints and the underlying OS.
 
-[Drozer](https://github.com/mwrlabs/drozer "Drozer on GitHub") is an Android security assessment framework that allows you to search for security vulnerabilities in apps and devices by assuming the role of a third-party app interacting with the other application's IPC endpoints and the underlying OS. The following section documents the steps necessary to install and use Drozer.
+###### Drozer Setup
 
-###### Installing Drozer
+You can refer to [drozer GitHub page](https://github.com/mwrlabs/drozer "Drozer on GitHub") (for Linux and Windows, for macOS please refer to this [blog post](https://blog.ropnop.com/installing-drozer-on-os-x-el-capitan/ "ropnop Blog - Installing Drozer on OS X El Capitan")) and the [drozer website](https://labs.mwrinfosecurity.com/tools/drozer/ "Drozer Website") for prerequisites and installation instructions.
```

Comment:
I restructured the sentence:

The installation instructions for drozer on Unix, Linux and Windows are explained in the [drozer Github page](https://github.com/mwrlabs/drozer "drozer GitHub page"). For [macOS this blog post](https://blog.ropnop.com/installing-drozer-on-os-x-el-capitan/ "Installing Drozer on OS X El Capitan") will be demonstrating all installation instructions. Another document with useful information might be the [official Drozer User Guide](https://labs.mwrinfosecurity.com/assets/BlogFiles/mwri-drozer-user-guide-2015-03-23.pdf "Drozer User Guide").
Useful resources can also be found in the [drozer GitHub page](https://github.com/mwrlabs/drozer "GitHub repo") and [Wiki](https://github.com/mwrlabs/drozer/wiki "drozer Wiki")

Reply:
slightly re-rephrased and added the resources as a list.